compiler/compiler.ts

```diff
@@ -2177,19 +2177,17 @@ export class Compiler {
             const id_buffer = typeof id == "string" ? hex2buffer(id, Pointer.MAX_POINTER_ID_SIZE, true) : id;
             const pointer_origin = (id_buffer[0]==BinaryCode.ENDPOINT || id_buffer[0]==BinaryCode.PERSON_ALIAS || id_buffer[0]==BinaryCode.INSTITUTION_ALIAS) ? <IdEndpoint> Target.get(id_buffer.slice(1,19), id_buffer.slice(19,21), id_buffer[0]) : null;
             
-            // TODO: enable
-            // const singleReceiver = 
-            //     (SCOPE.options.to instanceof Endpoint && SCOPE.options.to) ||
-            //     (SCOPE.options.to instanceof Disjunction && SCOPE.options.to.size == 1 && [...SCOPE.options.to][0] instanceof Endpoint && [...SCOPE.options.to][0])
+            const singleReceiver = 
+                (SCOPE.options.to instanceof Endpoint && SCOPE.options.to) ||
```

Comment:
Latter check of `SCOPE.options.to` in `SCOPE.options.to instanceof Endpoint && SCOPE.options.to` seems unnecessary

Reply:
Latter check of ` [...SCOPE.options.to][0]` is still unnecessary   `[...SCOPE.options.to][0] instanceof Endpoint && [...SCOPE.options.to][0]` 🚀